rl_runner.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2018 Google LLC
```

Comment:
I think this can just stay in rl_loop? we already have one layer of indirection with rl_loop => main; I don't think we need another.

Reply:
no, because importing stuff had side effects like evaluating hyperparams, bla bla bla.  This is how i actually run the code, and it shouldn't be bash.  I would like to have this layer not be needed as well, and in theory it shouldn't be, but in practice it is :)

Reply:
(most notably, the setup for cloud_logging is still side-effect-y)

Reply:
what sorts of side effects? We've been looking for a better hyperparam solution, so it'd be nice to know what kinds of issues we need to think about.

Reply:
i think the rewrite fixed the dual-graph-definition problem, which i think just leaves logging.  I really prefer the external runner though: would you be alright with it if i wrote it in bash?